app/config.js

```diff
@@ -41,4 +41,4 @@ window.OPENSHIFT_VERSION = {
   kubernetes: "dev-mode"
 };
 
-window.OPENSHIFT_CONSOLE_BASE_URL = "https://localhost:9000/dev-console";
+window.OPENSHIFT_CONSOLE_BASE_URL = "https://localhost:" + (window.DEV_SERVER_PORT || '9001');
```

Comment:
there is a projects section (though very rough) and the link should work. The sometimes working take you out of the showcase application. That said, I can put it back since it is customizable. It just seemed bad to me when I clicked the link and left the current app.